karma.conf.js

```diff
@@ -0,0 +1,88 @@
+/* eslint no-param-reassign:0, import/no-extraneous-dependencies:0 */
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = (config) => {
+  config.set({
+    basePath: '',
+
+    frameworks: ['mocha', 'sinon', 'chai'],
+
+    files: [
+      'test/_helpers/*.js',
+      'test/utils/*.js',
+      'test/components/*.jsx',
+    ],
+
+    webpack: {
+      externals: {
+        sinon: true,
+      },
+      plugins: [
+        // https://github.com/cheeriojs/cheerio/issues/836
+        new webpack.NormalModuleReplacementPlugin(/^\.\/package$/, (result) => {
+          if (/cheerio/.test(result.context)) {
+            result.request = './package.json';
+          }
+        }),
+      ],
+      module: {
+        loaders: [
+          {
+            test: /\.jsx?/,
+            loader: 'babel-loader',
+            include: [
+              path.join(__dirname, 'src'),
+              path.join(__dirname, 'test'),
+              path.join(__dirname, 'node_modules', 'airbnb-js-shims'),
```

Comment:
`require.resolve('airbnb-js-shims')`

in fact, i think this whole array could be `['./src', './test', 'airbnb-js-shims'].map(require.resolve)`?